buildstockbatch/gcp/gcp.py

```diff
@@ -884,7 +893,86 @@ def start_combine_results_job_on_cloud(self, results_dir, do_timeseries=True):
                     f"{self.postprocessing_job_console_url}",
                     exc_info=True,
                 )
-                break
+                return
+
+        # Monitor job/execution status, starting by polling the Job for an Execution
+        logger.info("Waiting for execution to begin...")
+        job_start_time = datetime.now()
+        jobs_client = run_v2.JobsClient()
+        job = jobs_client.get_job(name=self.postprocessing_job_name)
+        while not job.latest_created_execution:
+            time.sleep(1)
+            job = jobs_client.get_job(name=self.postprocessing_job_name)
+        execution_start_time = datetime.now()
+        logger.info(
+            f"Execution has started (after {str(execution_start_time - job_start_time)} "
+            "seconds). Waiting for execution to finish..."
+        )
+
+        # Have an execution; poll that for completion
+        fail_message = None
+        with tqdm.tqdm(
+            desc="Waiting for post-processing execution to complete", bar_format="{desc}: {elapsed}{postfix}"
+        ) as pp_tqdm:
+            spinner_states = ["|", "/", "-", "\\"]
+            spinner_state = 0
+
+            pp_tqdm.set_postfix_str("|")
+            executions_client = run_v2.ExecutionsClient()
+            execution_name = f"{self.postprocessing_job_name}/executions/{job.latest_created_execution.name}"
+            last_query_time = time.time()
+            while True:
+                # update spinner frequently...
+                time.sleep(0.25)
+                # ...but only actually query status every 10 sec
+                if time.time() - last_query_time > 10:
+                    # fetch and extract the status
+                    execution = executions_client.get_execution(name=execution_name)
+                    last_query_time = time.time()
+
+                    if execution.succeeded_count > 0:
+                        # Done!
+                        break
+                    elif execution.failed_count > 0:
+                        fail_message = "🟥 Post-processing execution failed."
+                        break
+                    elif execution.cancelled_count > 0:
+                        fail_message = "🟧 Post-processing execution canceled."
+                        break
+
+                spinner_state = (spinner_state + 1) % len(spinner_states)
+                pp_tqdm.set_postfix_str(spinner_states[spinner_state])
+                pp_tqdm.update()
+
+        if fail_message is not None:
+            # if logged within the tqdm block, the message ends up on the same line as the status
+            logger.warning(f"{fail_message} See {self.postprocessing_job_console_url} for more" " information")
```

Comment:
```suggestion
            logger.warning(f"{fail_message} See {self.postprocessing_job_console_url} for more information")
```

Reply:
Turns out this was the precommit formatter (it un-wrapped the lines, not consolidating the strings as it did so).